components/CostsCard.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <div class="card costs-card">
+    <!-- Todo: Make height dynamic. Matching header of time series. -->
+    <div class="card-header d-flex justify-content-between">
+      <div class="d-flex align-items-center">
+        <CIcon icon="cilChartPie" size="xl" class="mb-1 text-secondary" />
+        <h2 class="fs-5 m-0 ms-3 chart-header">
+          Losses
+        </h2>
+      </div>
+    </div>
+    <div id="costsCardBody" class="card-body">
+      <h3 id="totalHeading" class="mt-0 mb-0 fs-6">
+        TOTAL
+      </h3>
+      <div id="totalsContainer" ref="totalsContainer" class="d-flex flex-wrap gap-3 row-gap-0">
+        <div id="gdpContainer" class="d-flex gap-1">
+          <p id="gdpTotalCostPercent" class="mt-0 mb-0">
+            X.Y
+          </p>
+          <div id="gdpTotalCostPercentSymbolContainer">
+            <p id="gdpTotalCostPercentageSymbol" class="mb-0">
+              %
+            </p>
+            <p id="gdpTotalCostPercentReferent" class="mt-0 mb-0">
+              of GDP
+            </p>
+          </div>
+        </div>
+        <div id="usdContainer">
+          <div id="currency">
+            <p id="usdSymbol">
+              $
+            </p>
+            <p id="usdWord">
+              USD
+            </p>
+          </div>
+          <p id="totalCostPara" class="d-inline-block">
+            <span id="usdTotalCost">
+              <span>{{ totalCostAbbr?.amount }}</span>
+              <span id="totalCostUnit">
+                {{ totalCostAbbr?.unit }}
+              </span>
+            </span>
+          </p>
+        </div>
+      </div>
+      <CostsPie
+        id="costsPieContainer"
+        :hide-tooltips="hideTooltips"
+        :pie-size="pieSize"
+        :style="pieStyle"
+        @mouseleave="onMouseLeavePie"
+        @mouseover="() => { hideTooltips = false }"
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { abbreviateMillionsDollars } from "@/utils/money";
+import { CIcon } from "@coreui/icons-vue";
+
+const appStore = useAppStore();
+
+const totalsContainer = ref(null);
+const totalsContainerWidth = ref(0);
+
+const totalCostAbbr = computed(() => {
+  if (appStore.totalCost) {
+    return abbreviateMillionsDollars(appStore.totalCost?.value, 1, true);
+  } else {
+    return undefined;
+  }
+});
+
+// Highcharts does not automatically remove tooltips when the mouse leaves the chart area.
+const hideTooltips = ref(false);
+const onMouseLeavePie = () => {
+  setTimeout(() => {
+    hideTooltips.value = true;
+  }, 500);
+};
+
+// Fit the pie into the available width
+const pieSize = computed(() => {
+  return totalsContainerWidth.value * 0.6;
+});
```

Comment:
Now that the layout is fixed, would this not be simpler to do with css `width: 60%` in the `pieStyle`?

Reply:
Good point. In any case, this is removed in Anmol's costs table PR, in favour of two fixed width options depending on `appStore.largeScreen`.

Reply:
Ah ok, will just approve here, if it's on its way out anyway.